MAUI/Themes/Keys.md

```diff
@@ -10629,6 +10629,30 @@ This page lists the keys for each control and the element to which it is mapped
             <br/>
         </td>
     </tr>   
+     <tr>
+        <td>
+          SfPdfViewerControlBackgroundColor
+            <br/>
+            <br/>
+        </td> 
+        <td>
+           Background color of the SfPdfViewer control.
+            <br/>
```

Comment:
Did we take the description from source or created on own?

<img width="463" alt="image" src="https://github.com/user-attachments/assets/d84933ca-1f05-4fe8-b617-9fe804fe15e7" />

Reply:
description is created on my own sir

Reply:
Cross check whether this key is for document loading or page loading busy indicator
<img width="436" alt="image" src="https://github.com/user-attachments/assets/48ec2888-1f65-4231-8652-b271bf651183" />

Reply:
It is for both the document and page loading busy indicator